js/whitebit.js

```diff
@@ -885,19 +885,20 @@ module.exports = class whitebit extends Exchange {
         //         ],
         //     }
         //
-        // flattening orders and injecting the market
-        const keys = Object.keys (response);
-        const closedOrdersParsed = [];
-        for (let i = 0; i < keys.length; i++) {
-            const marketKey = keys[i];
-            const orders = response[marketKey];
+        const marketIds = Object.keys (response);
+        let results = [];
+        for (let i = 0; i < marketIds.length; i++) {
+            const marketId = marketIds[i];
+            const market = this.safeMarket (marketId);
```

Comment:
We can add a third argument here, that is the underscore `'_'` delimiter:

- https://github.com/carlosmiei/ccxt/blob/cfcf35eab4c72349094a9594f3e2d88c22facc61/js/base/Exchange.js#L1393

```JavaScript
            const market = this.safeMarket (marketId, undefined, '_');
```

We usually add it, if their symbols have a dash or an underscore inside the marketId.